rules/linux/process_creation/proc_creation_lnx_file_and_directory_discovery.yml

```diff
@@ -24,6 +24,13 @@ detection:
         Image|endswith: '/find'
     select_tree_execution:
         Image|endswith: '/tree'
+    select_findmnt_execution:
+        Image|endswith: '/findmnt'
+        CommandLine|contains:
+            - '-t nfs'
+            - '-t luks'
```

Comment:
```suggestion
```
Why force only nfs and luks ?
Find some sanbox with findmnt
- https://www.hybrid-analysis.com/sample/b33b415cbee7d64d4cbedda4c61d97437393b6095d53f597b5ee247b16591b6f/6661101ad9a9d3110608a0bf
- https://any.run/report/ed8fe6eb98c8a487c631dee11ddbe11c322e446666280f7b97844d259fdb10f5/10d65d81-d1ed-4034-ba9a-0a002a0b42c9

Reply:
I think it is better to remove the command-line.
All the other don't check it.

Reply:
If you think this work is better, I agree with you cleaning the command line. Thanks for the quick reply.